server/streamreader/librespot_stream.cpp

```diff
@@ -47,6 +47,7 @@ LibrespotStream::LibrespotStream(PcmListener* pcmListener, boost::asio::io_conte
     bool normalize = (uri_.getQuery("normalize", "false") == "true");
     bool autoplay = (uri_.getQuery("autoplay", "false") == "true");
     killall_ = (uri_.getQuery("killall", "true") == "true");
+    string volumectrl = uri_.getQuery("volumectrl", "log");
```

Comment:
is `log` the default in Librespot? Or should `uri_.getQuery("volumectrl", "")` be used and passed to Librespot if `!volumectrl.empty()`?